src/Geta.404Handler/Core/CustomRedirects/CustomRedirectCollection.cs

```diff
@@ -144,7 +144,16 @@ string RemoveSlash(string s)
             }
 
             var redirCopy = new CustomRedirect(cr);
-            var newUrl = AppendSlash(redirCopy.NewUrl);
+            var newUrl = string.Empty;
```

Comment:
You can use an expression `? :` together with the assignment.

Reply:
Changed the condition operator to include in variable assignment.